packages/gatsby-plugin-typography/src/gatsby-node.js

```diff
@@ -4,30 +4,41 @@ const os = require(`os`)
 
 // Write out a typography module to .cache.
 
-exports.onPreBootstrap = ({ store }, pluginOptions) => {
+exports.onPreBootstrap = ({ store, cache }, pluginOptions) => {
   const program = store.getState().program
 
   let module
   if (pluginOptions.pathToConfigModule) {
-    module = `module.exports = require("${
+    module = `export { default } from "${
```

Comment:
changing to ESM imports here might have some side effects. Is this needed?

Reply:
I would like to dial back those changes here (if possible)

exporting `default` is different than using `module.exports` and might break some setups

Reply:
> changing to ESM imports here might have some side effects. Is this needed?

I admit, I didn't think that it would cause issues, since these files are all processed by Webpack anyway.  

> exporting `default` is different than using `module.exports` and might break some setups

🤔 You're right; I guess it would cause problems if somebody is using es5 `module.exports = <typography>` in their custom typography file.  Thank you for pointing it out.

Changes have been reversed.